includes/emails/class-wc-email-cancelled-order.php

```diff
@@ -90,7 +90,8 @@ public function trigger( $order_id, $order = false ) {
 			}
 
 			if ( $this->is_enabled() && $this->get_recipient() ) {
-				$this->send( $this->get_recipient(), $this->get_subject(), $this->get_content(), $this->get_headers(), $this->get_attachments() );
+				$result = ( $this->send( $this->get_recipient(), $this->get_subject(), $this->get_content(), $this->get_headers(), $this->get_attachments() ) ) ? true : false;
```

Comment:
By default `WC_Email->send()` returns boolean, so there's no need to any extra code:

```suggestion
				$result = $this->send( $this->get_recipient(), $this->get_subject(), $this->get_content(), $this->get_headers(), $this->get_attachments() );
```